src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -242,14 +254,13 @@ public HttpApiHandler(DurableTaskExtension config, TraceWriter traceWriter)
                     return request.CreateResponse(HttpStatusCode.Gone);
             }
 
-            string mediaType = request.Content.Headers.ContentType?.MediaType;
-            if (!string.IsNullOrEmpty(mediaType) && 
-                !string.Equals(mediaType, "application/json", StringComparison.OrdinalIgnoreCase))
+            var mediaType = request.Content.Headers.ContentType?.MediaType;
+            if (!string.Equals(mediaType, "application/json", StringComparison.OrdinalIgnoreCase))
             {
                 return request.CreateErrorResponse(HttpStatusCode.BadRequest, "Only application/json request content is supported");
             }
 
-            string stringData = await request.Content.ReadAsStringAsync();
+            var stringData = await request.Content.ReadAsStringAsync();
```

Comment:
I prefer not using `var` in place of primitive type names. I generally only use it when the type is already specified in the right-hand expression - e.g. `var x = new Foo()`. Using `var` carefully is good to ensure that the code is readable and as easy to understand as possible.

Reply:
Got it. I will follow this approach from now on.